extractor/src/main/java/org/schabi/newpipe/extractor/stream/AudioStream.java

```diff
@@ -33,16 +33,6 @@ public final class AudioStream extends Stream {
 
     private final int averageBitrate;
 
-    // Fields for DASH
-    private int itag = ITAG_NOT_AVAILABLE_OR_NOT_APPLICABLE;
-    private int bitrate;
-    private int initStart;
-    private int initEnd;
-    private int indexStart;
-    private int indexEnd;
-    private String quality;
-    private String codec;
```

Comment:
Hi, I was the one who added these fields originally, they are used in Piped's backend for providing sidX boxes for streaming with DASH.

Here's how an audio representation is created by a client: https://github.com/TeamPiped/Piped/blob/8ee2a1c20737ce9db84186e00849b405d6c097af/src/utils/DashUtils.js#L99-L117